packages/components/src/tooltip/stories/index.js

```diff
@@ -17,79 +6,80 @@ import Tooltip from '../';
 export default {
 	title: 'Components/ToolTip',
 	component: Tooltip,
+	argTypes: {
+		delay: { control: 'number' },
+		position: {
```

Comment:
Should we default to `top center` as we did for the previous version of the story that used knobs?

```suggestion
		position: {
			defaultValue: 'top center',
```

Reply:
Same here.